hack/update-generated-protobuf.sh

```diff
@@ -31,19 +31,13 @@ function prereqs() {
   fi
   kube::build::ensure_docker_daemon_connectivity || return 1
 
-  KUBE_ROOT_HASH=$(kube::build::short_hash "${HOSTNAME:-}:${REPO_DIR:-${KUBE_ROOT}}/go-to-protobuf")
+  KUBE_ROOT_HASH=$(kube::build::short_hash "${HOSTNAME:-}:${REPO_DIR:-${KUBE_ROOT}}")
```

Comment:
@smarterclayton This makes a hash that is never ever cleaned up.  I think this might be related to trying to build releases, which already run under docker?